src/freezegun.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+import datetime
+
+import time_machine
+
+
+def freeze_time(t: str | datetime.datetime = "2023-09-13 01:02:00") -> time_machine.travel:
+    return time_machine.travel(t, tick=False)
```

Comment:
this is a proof-of-concept -- to get this over the line I'll be moving this (and the imports) into testutils